src/scenes/Projects/Overview/ProjectOverview.tsx

```diff
@@ -273,6 +283,40 @@ export const ProjectOverview: FC = () => {
               }
               className={classes.pushPinIcon}
             />
+            {(!projectOverviewData ||
+              projectOverviewData.project.presetInventory.canEdit) && (
+              <Tooltip title="This indicates the project/language(s) will be exposed to major investors to directly fund.">
```

Comment:
Instead of having a new component under the `Projects` directory, added the preset inventory icon directly into the detail component.